custom-metrics-stackdriver-adapter/pkg/adapter/coreprovider/client.go

```diff
@@ -0,0 +1,86 @@
+package coreprovider
+
+import (
+	translator "github.com/GoogleCloudPlatform/k8s-stackdriver/custom-metrics-stackdriver-adapter/pkg/adapter/translator"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/klog"
+	"sigs.k8s.io/metrics-server/pkg/api"
+)
+
+const (
+	cpuMetricKind = "CUMULATIVE"
+	ramMetricKind = "GUAGE"
+
+	containerCpuMetricName = "kubernetes.io/container/cpu/core_usage_time"
+	containerRamMetricName = "kubernetes.io/container/memory/used_bytes"
+	nodeCpuMetricName      = "kubernetes.io/node/cpu/core_usage_time"
+	nodeRamMetricName      = "kubernetes.io/node/memory/used_bytes"
+)
+
+type stackdriverCoreClient struct {
+	translator *translator.Translator
+}
+
+func newClient(translator *translator.Translator) *stackdriverCoreClient {
+	return &stackdriverCoreClient{translator}
+}
+
+// TODO(holubowicz): handle a case when len(resourceNames) > oneOfMax
+func (p *stackdriverCoreClient) getContainerCpu(podsNames []string) (map[string]map[string]resource.Quantity, map[string]api.TimeInfo, error) {
```

Comment:
CPU is an abbreviation, should be
```suggestion
func (p *stackdriverCoreClient) getContainerCPU(podsNames []string) (map[string]map[string]resource.Quantity, map[string]api.TimeInfo, error) {
```